features/polyExplorer/src/styles.css

```diff
@@ -1,7 +1,11 @@
 :root {
```

Comment:
Add the colors to poly-looks colors.css and use them from there. This file should not be used anymore. (You may need to merge the new release branch)

Reply:
So cool thanks :)